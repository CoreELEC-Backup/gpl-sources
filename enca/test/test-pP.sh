#! /bin/sh
# Purpose: test whether -p, -P and default file prefixing works
. $srcdir/setup.sh
TEST_TEXT=$srcdir/cs-s.iso88592
OPTS="-e -L cs"
$ENCA $OPTS $TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo AAA >>$TESTNAME.actual
$ENCA $OPTS -p $TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo BBB >>$TESTNAME.actual
$ENCA $OPTS -P $TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo CCC >>$TESTNAME.actual
$ENCA $OPTS <$TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo DDD >>$TESTNAME.actual
$ENCA $OPTS -p <$TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo EEE >>$TESTNAME.actual
$ENCA $OPTS -P <$TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo FFF >>$TESTNAME.actual
$ENCA $OPTS - <$TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo GGG >>$TESTNAME.actual
$ENCA $OPTS -p - <$TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo HHH >>$TESTNAME.actual
$ENCA $OPTS -P - <$TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo III >>$TESTNAME.actual
$ENCA $OPTS $TEST_TEXT $TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo JJJ >>$TESTNAME.actual
$ENCA $OPTS -p $TEST_TEXT $TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
echo KKK >>$TESTNAME.actual
$ENCA $OPTS -P $TEST_TEXT $TEST_TEXT | sed -e "s#^$srcdir/##" >>$TESTNAME.actual || DIE=1
. $srcdir/finish.sh
