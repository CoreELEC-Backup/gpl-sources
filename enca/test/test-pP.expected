ISO-8859-2
AAA
cs-s.iso88592: ISO-8859-2
BBB
ISO-8859-2
CCC
ISO-8859-2
DDD
STDIN: ISO-8859-2
EEE
ISO-8859-2
FFF
ISO-8859-2
GGG
STDIN: ISO-8859-2
HHH
ISO-8859-2
III
cs-s.iso88592: ISO-8859-2
cs-s.iso88592: ISO-8859-2
JJJ
cs-s.iso88592: ISO-8859-2
cs-s.iso88592: ISO-8859-2
KKK
ISO-8859-2
ISO-8859-2
